Cargo.toml

```diff
@@ -30,8 +30,8 @@ revm-primitives = "1.1"
 reth-rlp = { git = "https://github.com/paradigmxyz/reth.git", tag = "v0.1.0-alpha.7" }
 
 # Kakarot deps
-kakarot-rpc-core = { git = "https://github.com/kkrt-labs/kakarot-rpc.git", rev = "0894457" }
-kakarot-test-utils = { git = "https://github.com/kkrt-labs/kakarot-rpc.git", rev = "0894457" }
+kakarot-rpc-core = { git = "https://github.com/kkrt-labs/kakarot-rpc.git", rev = "ae5e220" }
```

Comment:
this is the bump for steps right?

Reply:
I don't think so, I think it was a small bump by me when I thought it changed something (it doesn't)